ldi-core/ldi-infra-sql/src/main/resources/META-INF/persistence.xml

```diff
@@ -10,12 +10,11 @@
         <properties>
 
             <property name="hibernate.dialect" value="org.sqlite.hibernate.dialect.SQLiteDialect"/>
-            <property name="hibernate.hbm2ddl.auto" value="update"/>
+<!--            <property name="hibernate.hbm2ddl.auto" value="update"/>-->
```

Comment:
Is there a reason this remains as a comment?

Reply:
Aside I only forgot the remove it after everything works, there is not 🙈